docs/feature_combo.md

```diff
@@ -71,26 +92,118 @@ void process_combo_event(uint16_t combo_index, bool pressed) {
 }
 ```
 
-This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V.  But you could change this to do stuff like change layers, play sounds, or change settings.
-
-## Additional Configuration
-
-If you're using long combos, or even longer combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing.
-
-In this case, you can add either `#define EXTRA_LONG_COMBOS` or `#define EXTRA_EXTRA_LONG_COMBOS` in your `config.h` file.
-
-You may also be able to enable action keys by defining `COMBO_ALLOW_ACTION_KEYS`.
+This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V. But you could change this to do stuff like play sounds or change settings.
 
-## Keycodes 
+# Advanced Configuration
+These configuration settings can be set in your `config.h` file.
 
-You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. 
+## Keycodes
+You can enable, disable and toggle the Combo feature on the fly. This is useful if you need to disable them temporarily, such as for a game. The following keycodes are avalible for use in your `keymap.c`
 
 |Keycode   |Description                      |
 |----------|---------------------------------|
 |`CMB_ON`  |Turns on Combo feature           |
 |`CMB_OFF` |Turns off Combo feature          |
 |`CMB_TOG` |Toggles Combo feature on and off |
 
+## Combo Term
+By default, the timeout for the Combos to be recognized is set to 50ms. This can be changed if accidental combo misfires are happening or if you're having difficulties pressing keys at the same time. For instance `#define COMBO_TERM 40` would set the time out period for combos to 40ms.
+
+## Long Combos
+If you're using long combos, or you have a lot of overlapping combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing. In this case, you can configure the size of the data structure used. Be aware, larger combo sizes will increase memory usage!
+
+| Keys | Define to be set     |
+|------|----------------------|
+| 8    | QMK Default          |
+| 16   | `#define EXTRA_LONG_COMBOS`  |
+| 32   | `#define EXTRA_EXTRA_LONG_COMBOS`|
+
+## Modifier Combos
+If a combo resolves to a Modifier, the window for processing the combo can be extended independent from normal combos. By default this is disabled but can be enabled with `#define COMBO_MUST_HOLD_MODS`, and the time window can be configured with `#define COMBO_MOD_TERM 150` (default: 200). With `COMBO_MUST_HOLD_MODS` you cannot tap the combo any more which makes the combo less prone to misfires.
+
+## Per Combo Timing and Holding
+Instead of using a blanket `COMBO_TERM` window for every combo, per-combo timings can be set. In order to use this feature the following config options and functions need to be defined. Coming up with useful timings and configuration is left as an exercise for the reader.
```

Comment:
```suggestion
Instead of using a blanket `COMBO_TERM` window for every combo, per-combo timings can be set. In order to use this feature, the following configuration options and functions need to be defined. Coming up with useful timings and configuration is left as an exercise for the reader.
```